src/style.css

```
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir-Book", serif;
+}
+
+body {
+  font-weight: 300;
+}
+
+.search-bar::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-bar-big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-bar-small::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-bar {
+  width: 100%;
+  color: rgba(61, 78, 97, 1);
+  border: 1px solid hsl(210, 25%, 91%);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar-big {
```

It is a modificator, so let`s keep BEM syntax
```suggestion
.search-bar--big {
```